README.md

```diff
@@ -0,0 +1,5 @@
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```

Comment:
This comments belong to the original template you used. They are not supposed to show on your repo.

Reply:
I recommend you make your comments more descriptive and try to  be more of suggestive than instructive.🤗 
You can see a more robust explanation in the first comment🤗